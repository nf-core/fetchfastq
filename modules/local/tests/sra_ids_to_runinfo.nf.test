nextflow_process {

    name "Test Process SRA_IDS_TO_RUNINFO"
    script "modules/local/sra_ids_to_runinfo.nf"
    process "SRA_IDS_TO_RUNINFO"
    tag "sra_ids_to_runinfo"
    tag "modules_local"

    test("Should run without failures") {

        when {
            params {
                outdir   = "$outputDir"
            }
            process {
                """
                input[0] = 'SRR9984183'
                input[1] = ''
                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out).match() },
            )
        }

    }

}
