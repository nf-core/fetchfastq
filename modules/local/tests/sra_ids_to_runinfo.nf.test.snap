{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "SRR9984183.runinfo.tsv:md5,51cba5fab4d6f0c0386e4397672240b9"
                ],
                "1": [
                    "versions.yml:md5,1c14442e9b494b586eafe41e77300fae"
                ],
                "tsv": [
                    "SRR9984183.runinfo.tsv:md5,51cba5fab4d6f0c0386e4397672240b9"
                ],
                "versions": [
                    "versions.yml:md5,1c14442e9b494b586eafe41e77300fae"
                ]
            }
        ],
        "timestamp": "2023-07-06T13:16:51+0000"
    }
}